crc.py

```diff
@@ -488,7 +489,11 @@ def argument_parser():
     t.set_defaults(func=table)
 
     c = subparsers.add_parser('checksum', help='Calculate checksum(s) for the specified input(s)')
-    c.add_argument('inputs', nargs='*', type=argparse.FileType('r'), default=[sys.stdin],
+    try:
+        stdinraw=sys.stdin.buffer
+    except AttributeError:
+        stdinraw=io.BytesIO(sys.stdin.read().encode('ascii'))
+    c.add_argument('inputs', nargs='*', type=argparse.FileType('rb'), default=[stdinraw],
```

Comment:
I rather would go with adjusting the mock, I don't think there is a scenario where the real `sys.stdin` does not have a buffer.

Reply:
Even passing the `-u` flag to the python interpreter only seem to affect `sys.stdout` and `sys.stderr`.
I think if we ever see the extra test is needed we add it. For now I would say `yagni` :D

Reply:
More iagni, because it was added for /your/ unit tests...

Reply:
yes that's why I suggest to rather fix the test rather than adjust the code. like in the example PR I have created.
I don't think it does make any sense to add extra code to the actual project to work around "test issues".

Reply:
https://github.com/Nicoretti/crc/pull/36/files